src/main/python/covata/delta/api/requestsclient.py

```diff
@@ -16,32 +16,33 @@
 
 import requests
 
-from covata.delta.util import LogMixin
-from covata.delta.api import ApiClient
+from covata.delta import ApiClient, LogMixin, crypto
+from .signer import RequestsSigner
 
 
 class RequestsApiClient(ApiClient, LogMixin):
     def register_identity(self, external_id=None, metadata=None):
-        crypto = self._crypto_service
+        keystore = self.keystore
         signing_private_key = crypto.generate_key()
         crypto_private_key = crypto.generate_key()
 
         signing_public_key = signing_private_key.public_key()
         crypto_public_key = crypto_private_key.public_key()
 
-        body = dict(signingPublicKey=crypto.serialized(signing_public_key),
-                    cryptoPublicKey=crypto.serialized(crypto_public_key),
-                    externalId=external_id,
-                    metadata=metadata)
+        body = dict(
+            signingPublicKey=crypto.serialize_public_key(signing_public_key),
+            cryptoPublicKey=crypto.serialize_public_key(crypto_public_key),
+            externalId=external_id,
+            metadata=metadata)
 
         response = requests.post(
             url=self.DELTA_URL + self.RESOURCE_IDENTITIES,
             json=dict((k, v) for k, v in body.items() if v is not None))
 
         identity_id = response.json()['identityId']
 
-        crypto.save(signing_private_key, identity_id + ".signing.pem")
-        crypto.save(crypto_private_key, identity_id + ".crypto.pem")
+        keystore.save(signing_private_key, identity_id + ".signing.pem")
```

Comment:
I feel like this would read better if the filename is not exposed as a parameter.

- keystore.save_encryption_key(identity_id)
- keystore.save_signing_key(identity_id)

Or maybe better still (similar to Java, where you can't save one without the other, so both are paramters to a single method):

- keystore.store_keys(identity_id, signing_key, encryption_key)